storage/innobase/fsp/fsp0fsp.cc

```diff
@@ -37,12 +37,16 @@ Created 11/29/1995 Heikki Tuuri
 #include "btr0sea.h"
 #include "dict0boot.h"
 #include "log0log.h"
+#include "dict0load.h"
 #include "dict0mem.h"
-#include "fsp0types.h"
+#include "btr0pcur.h"
+#include "trx0sys.h"
```

Comment:
`trx0sys.h` includes many headers, such as `buf0buf.h`, `fil0fil.h` and `ut0byte.h`. The redundant headers should be removed.